From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: thomasclark <tjclark@wpi.edu>
Date: Wed, 2 Jul 2014 12:14:49 -0400
Subject: [PATCH 0153/6262] Fixed periodic voltage status message

Previously, the voltage was requested as a percentage and decoded as
a number of volts, which resulted in values being scaled wrong. The
correct message is requested now.

Change-Id: I5e81c7a4a2e2698f2e8a84ba747217e0e14f7676
---
 wpilibc/wpilibC++/lib/CANJaguar.cpp                             | 2 +-
 .../src/main/java/edu/wpi/first/wpilibj/CANJaguar.java          | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wpilibc/wpilibC++/lib/CANJaguar.cpp b/wpilibc/wpilibC++/lib/CANJaguar.cpp
index 5c1302f61388a325cb8b7fa99bd880d02587580a..aabc7253f40ed0d682f20d7772a6d871c0597735 100644
--- a/wpilibc/wpilibC++/lib/CANJaguar.cpp
+++ b/wpilibc/wpilibC++/lib/CANJaguar.cpp
@@ -504,7 +504,7 @@ void CANJaguar::setupPeriodicStatus() {
 	// temperature.
 	static const uint8_t kMessage0Data[] = {
 		LM_PSTAT_VOLTBUS_B0, LM_PSTAT_VOLTBUS_B1,
-		LM_PSTAT_VOLTOUT_B0, LM_PSTAT_VOLTOUT_B1,
+		LM_PSTAT_VOUT_B0, LM_PSTAT_VOUT_B1,
 		LM_PSTAT_CURRENT_B0, LM_PSTAT_CURRENT_B1,
 		LM_PSTAT_TEMP_B0, LM_PSTAT_TEMP_B1
 	};
diff --git a/wpilibj/wpilibJava/src/main/java/edu/wpi/first/wpilibj/CANJaguar.java b/wpilibj/wpilibJava/src/main/java/edu/wpi/first/wpilibj/CANJaguar.java
index 3110745642792e73334d1ea50dd8627ecb3d6fcc..9cd5262b12602af3d44b1ac353e65aa89b3738ae 100644
--- a/wpilibj/wpilibJava/src/main/java/edu/wpi/first/wpilibj/CANJaguar.java
+++ b/wpilibj/wpilibJava/src/main/java/edu/wpi/first/wpilibj/CANJaguar.java
@@ -1749,7 +1749,7 @@ public class CANJaguar implements MotorSafety, PIDOutput, SpeedController, LiveW
 		// temperature.
 		final byte[] kMessage0Data = new byte[] {
 			CANJNI.LM_PSTAT_VOLTBUS_B0, CANJNI.LM_PSTAT_VOLTBUS_B1,
-			CANJNI.LM_PSTAT_VOLTOUT_B0, CANJNI.LM_PSTAT_VOLTOUT_B1,
+			CANJNI.LM_PSTAT_VOUT_B0, CANJNI.LM_PSTAT_VOUT_B1,
 			CANJNI.LM_PSTAT_CURRENT_B0, CANJNI.LM_PSTAT_CURRENT_B1,
 			CANJNI.LM_PSTAT_TEMP_B0, CANJNI.LM_PSTAT_TEMP_B1
 		};
